marxs/optics/polarization.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from ..math.pluecker import *
+
+
+def polarization_vectors(dir_array, angles):
+	n = len(angles)
+	polarization = np.zeros((n, 4))
+	x = np.array([1., 0., 0.])
+	y = np.array([0., 1., 0.])
+	for i in range(0, n):
```

Comment:
If we can, I'd like to write this in array notation and avoid the for loop, see above...
But you don't have to do that now if it's more than 5 min of work - we can merge this like it is now, and leave a note in the code that this is a good place to start to optimize if it's too slow. (It might not be too slow and in that case why bother?)